gunicorn - RuntimeError: Working outside of application context.

In a fresh setup from trunk, using gunicorn,
superset does not work, not even the main page.
instead "superset run" works ok.

Command Line:
gunicorn \
      -w 8 \
      -k gevent \
      --timeout 120 \
      -b  0.0.0.0:5000 \
      --limit-request-line 0 \
      --limit-request-field_size 0 \
      superset:app

Error when trying http://127.0.0.1:5000/

[2019-12-01 12:08:34 +0100] [11510] [ERROR] Error handling request /
Traceback (most recent call last):
  File "/home/user/software/venvSS/lib/python3.6/site-packages/gunicorn/workers/base_async.py", line 55, in handle
    self.handle_request(listener_name, req, client, addr)
  File "/home/user/software/venvSS/lib/python3.6/site-packages/gunicorn/workers/ggevent.py", line 143, in handle_request
    super().handle_request(listener_name, req, sock, addr)
  File "/home/user/software/venvSS/lib/python3.6/site-packages/gunicorn/workers/base_async.py", line 106, in handle_request
    respiter = self.wsgi(environ, resp.start_response)
  File "/home/user/software/venvSS/lib/python3.6/site-packages/werkzeug/local.py", line 376, in <lambda>
    __call__ = lambda x, *a, **kw: x._get_current_object()(*a, **kw)
  File "/home/user/software/venvSS/lib/python3.6/site-packages/werkzeug/local.py", line 307, in _get_current_object
    return self.__local()
  File "/home/user/software/venvSS/lib/python3.6/site-packages/flask/globals.py", line 52, in _find_app
    raise RuntimeError(_app_ctx_err_msg)
RuntimeError: Working outside of application context.

This typically means that you attempted to use functionality that needed
to interface with the current application object in some way. To solve
this, set up an application context with app.app_context().  See the
documentation for more information.
[2019-12-01 12:08:35 +0100] [11508] [ERROR] Error handling request /favicon.ico
Traceback (most recent call last):
  File "/home/user/software/venvSS/lib/python3.6/site-packages/gunicorn/workers/base_async.py", line 55, in handle
    self.handle_request(listener_name, req, client, addr)
  File "/home/user/software/venvSS/lib/python3.6/site-packages/gunicorn/workers/ggevent.py", line 143, in handle_request
    super().handle_request(listener_name, req, sock, addr)
  File "/home/user/software/venvSS/lib/python3.6/site-packages/gunicorn/workers/base_async.py", line 106, in handle_request
    respiter = self.wsgi(environ, resp.start_response)
  File "/home/user/software/venvSS/lib/python3.6/site-packages/werkzeug/local.py", line 376, in <lambda>
    __call__ = lambda x, *a, **kw: x._get_current_object()(*a, **kw)
  File "/home/user/software/venvSS/lib/python3.6/site-packages/werkzeug/local.py", line 307, in _get_current_object
    return self.__local()
  File "/home/user/software/venvSS/lib/python3.6/site-packages/flask/globals.py", line 52, in _find_app
    raise RuntimeError(_app_ctx_err_msg)
RuntimeError: Working outside of application context.

This typically means that you attempted to use functionality that needed
to interface with the current application object in some way. To solve
this, set up an application context with app.app_context().  See the
documentation for more information.

--- a/docs/installation.rst
+++ b/docs/installation.rst
@@ -220,7 +220,7 @@ setup known to work well in production: ::
         --limit-request-line 0 \
         --limit-request-field_size 0 \
         --statsd-host localhost:8125 \
-        superset:app
+        "superset.app:create_app()"
 
 Refer to the
 `Gunicorn documentation <https://docs.gunicorn.org/en/stable/design.html>`_

